### DISCUSSION TITLE
Trouble in egs_view

### DISCUSSION THREAD

**Discussion by Wanggl180328** [nrc-cnrc/EGSnrc#1166] [Questions] [answered]
I am a new egs user and just followed the getting started. After doing myapp -i slab.egsinp, the information are shown in the command line. But no picture are shown, I donot know if it is unusual. Then I started next step egs_view slab.egsinp slab.ptracks
, but it has an error like egs_view: command not found. I also tried use gui but also no picture is shown.

**Comment by rtownson** [comment]
Did you install using the installation GUI? What operating system?

If the installer placed egs_view as a shortcut on your desktop, you could just launch it there. It will then prompt you to select the .egsinp file.

**Reply by Wanggl180328** [reply]
Thanks for you concern. I  installed a virtual machine (Lubuntu) on windows using Virtual box. The codes are running on the Lubuntu operating system. Doing the installation and the getting started pdf, I did not find the process leading me to install GUI and no shortcut is shown in the desktop. But I can directly run the egs_gui in the command line, enven though I did not find egs_gui's shortcut.

**Reply by rtownson** [reply, marked as answer]
Download https://github.com/nrc-cnrc/EGSnrc/releases/download/v2023/EGSnrc-guis-linux.tar.gz, and extract it. Place the `pieces/linux folder` into `HEN_HOUSE/pieces`, and place the `egs++/dso/linux-static` folder into `HEN_HOUSE/egs++/dso/`. 

Try launching egs_view with:
```
$HEN_HOUSE/pieces/linux/egs_view_64
```

If that gives an error, try:
```
export LD_LIBRARY_PATH=$HEN_HOUSE/egs++/dso/linux-static; $HEN_HOUSE/pieces/linux/egs_view_64
```

**Reply by Wanggl180328** [reply]
Thank you very much for your help，then it works well.

**Comment by Laymandamn** [comment]
作者你好，我也是egs新用户使用者，正在通过学习手册进行学习，我使用的是windows系统进行学习的，当我进行到egs_view slab.egsinp slab.ptracks这一步时，我发现我显示出来的图像是一个平面和一个三维坐标系，那些红色的粒子轨迹并没有显示出来，这使我非常苦恼。当我用git bash 对egs_view slab.ptracks进行操作时，会显示segmentation fault，并出现闪退现象。还有另一个问题就是当我运行到 mingw32-make这一步时出现了一个错误/usr/bin/sh: -c: line 2: syntax error: unexpected end of file，这是不是对上一个问题出现的罪魁祸首

**Reply by rtownson** [reply]
Hello, please post this as a new discussion. Include your installation log, found in `HEN_HOUSE/log`. We need to ensure that you're able to compile egs_app correctly, and then that egs_app runs and produces the .ptracks file.
